hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/fs/TestDFSIO.java

```diff
@@ -289,12 +297,43 @@ public void testTruncate() throws Exception {
     bench.analyzeResult(fs, TestType.TEST_TYPE_TRUNCATE, execTime);
   }
 
+  private class ControlFileCreateTask implements Callable<Void> {
```

Comment:
There is no return value here, it is more suitable to use Runnable.

Reply:
@zhangshuyan0 Thanks sir, have updated here.